shell/platform/darwin/ios/platform_view_ios.h

```diff
@@ -98,6 +103,16 @@ class PlatformViewIOS final : public PlatformView {
     return platform_message_handler_;
   }
 
+  /**
+   * Gets the accessibility bridge created in this platform view.
+   */
+  AccessibilityBridge* GetAccessibilityBridge() { return accessibility_bridge_.get(); }
```

Comment:
These are public for testing only, i am not sure how to best test the change without exposing these

Reply:
This is fine for now; refactoring this class is on my to do list for a rainy day, but please add a note pointing out this is only for testing so we can clean it up in the refactor.